debian/watch

```diff
@@ -0,0 +1,4 @@
+# please also check http://pypi.debian.net/txfixtures/watch
+version=3
+opts=uversionmangle=s/(rc|a|b|c)/~$1/,pgpsigurlmangle=s/$/.asc/ \
+https://pypi.debian.net/txfixtures/txfixtures-(.+)\.(?:zip|tgz|tbz|txz|(?:tar\.(?:gz|bz2|xz)))
```

Comment:
Do we know what this file is for?

Reply:
`watch` file is not needed during the build process, but it rather used after it.  There is a `uscan` program that parses these files and notifies if any new versions of package is available.  

I don't think it will affect anything. Also, I think the whole txfixtures package was under Debian org earlier.